tests/testthat/tests_imports.R

```diff
@@ -114,7 +111,8 @@ test_that("External importWaterML1 test", {
   siteNumber <- c('01480015',"04085427") #one site seems to have lost it's 2nd dd
   obs_url <- constructNWISURL(siteNumber,c("00060","00010"),startDate,endDate,'dv')
   data <- importWaterML1(obs_url)
-  # saveRDS(data, "rds/twoWML1.rds")
+  
+  expect_false(any(data$X_00010_00003) )
```

Comment:
Looks like there is mean temperature in the data...what's this test for?

Reply:
not sure what is happening here, this is supposed to be this: 
https://github.com/wdwatkins/dataRetrieval/blob/upstreamTest/tests/testthat/tests_imports.R#L179

Checking to see that there aren't any of the no data values left

Reply:
hmm, this must have been left from when I started to write this test here and then moved it further down.  I just assumed the error was happening further down at ~#178 :man_facepalming: